docs/bokeh/source/_static/custom.css

```diff
@@ -166,10 +166,34 @@ div.highlight pre {
   color: #ddd;
 }
 
-/* Floating gallery thumbnails */
-ul.gallery li {
-  list-style-type: none;
-  float:left
+div.bk-gallery {
+    width: 100%;
+    margin: 0px 0px 20px 0px;
+    padding-left: 50px;
+    padding-right: 50px;
+    justify-content: space-between;
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+    gap: 15px;
+}
+
+div.bk-thumbnail {
+    background: transparent;
+    border-radius: 7px;
+    box-shadow: 0 0 3px #444;
+    position: relative;
+    box-sizing: border-box;
+    width: 100%;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 10px;
+}
+
+div.bk-thumbnail-title {
+    font-weight: bold;
+    font-size: 12px;
 }
 
 img.gallery {
```

Comment:
This selector forces the gallery thumbnails to be 200x200. The actual size (modulo retina "@2x" versions) is 300x300. Not sure if we should keep this or not. This will cause some downsampling but at least on my screen 200x200 seems to be a comfortable size. 

In any case if we keep it it should change to `img.bk-gallery` and then also the width/height here should probably be coordinated with the `grid-template-columns` size above